src/server/routes/conversions.js

```diff
@@ -68,7 +69,7 @@ router.post('/edit', async (req, res) => {
 	const validatorResult = validate(req.body, validConversion);
 	if (!validatorResult.valid) {
 		log.warn(`Got request to edit conversions with invalid conversion data, errors:${validatorResult.errors}`);
-		res.status(400);
+		failure(res, 400, "Error(s): " + validatorResult.errors.toString());
```

Comment:
As is often the case for the related log message, it would be nice to include what was happening in the message returned to the user. For example, this could become: \
Got request to edit conversions with invalid conversion data, errors:${validatorResult.errors.toString() \
This applies to all the new messages in the PR where the needed change only applies to only some.